cogs/tiles.py

```diff
@@ -17,13 +18,8 @@
 class Tiles(Cog):
     """Send game tiles to Discord."""
 
-    @command()
-    async def tile(self, ctx: Context, *args, smalltile=False):
-        """Send the tile for the named Qud blueprint.
-
-        Optional postfix: recolor color1 color2
-        Append this to the command to recolor the tile with color1 as the tile color and color2
-        as the detailcolor.  color1 and color2 should be Qud color codes like 'w' and 'c'."""
+    @staticmethod
```

Comment:
Since this is now a static method it can be broken out from the class entirely and become a helper function. I would probably still leave it in this file though.

It was a good idea to break out the logic from the commands.

Reply:
Good idea - moved to `TilesHelper`